index.php

```diff
@@ -1 +1,44 @@
+<html lang="en">
+<head>
+    <title>Crep Collector</title>
+    <link rel="stylesheet" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com" rel="stylesheet">
+</head>
+<body>
+<nav>
+    <h1>Trainer Tracker</h1>
+</nav>
+
+
 <?php
+
+$db = new PDO('mysql:host=db; dbname=shoes', 'root', 'password');
+$db->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$query = $db->prepare('SELECT `brand`, `name`, `color`, `cost`, `image` FROM `shoes`');
+$query->execute();
+$results = $query->fetchAll();
```

Comment:
Could use require once instead?

Reply:
Jo's suggestion is a good idea! It would be handy to split this into a separate db.php file.